website/docs/r/is_security_group_network_interface_attachment.html.markdown

```diff
@@ -18,6 +18,7 @@ resource "ibm_is_security_group_network_interface_attachment" "sgnic" {
   network_interface = "6d6128aa-badc-45c4-bb0e-7c2c1c47be55"
 }
 ```
+**Note** This resource is deprecated. Use `ibm_is_security_group_target` to attach a network interface to a security group
```

Comment:
@hkantare Should we add a date to the deprecation notice as a reminder to remove these items in 6 months time frame?

Reply:
https://www.terraform.io/docs/extend/best-practices/deprecations.html#provider-data-source-or-resource-removal

Please follow this procedure for deprecation and removal